py/client/examples/csv_downloader.py

```diff
@@ -5,12 +5,8 @@
 
 from requests import get
 
-
-def download_csv(url: str, file_name=None) -> str:
-    """ download a CSV located at the supplied URL and return the file name. """
-    if not file_name:
-        file_name = "sample.csv"
-
+def download_file(url: str, file_name: str) -> str:
```

Comment:
This function worked okay, but since we're downloading parquet files instead the name/default didn't seem appropriate. (And the function works equally well for any filetype, really)